src/app/global/components/table-custom/table-custom.component.html

```diff
@@ -0,0 +1,32 @@
+<body>
+
+  <p-table [value]="elementos" styleClass="p-datatable-striped"
+    responsiveLayout="stack"
+    selectionMode="single"
+    [breakpoint]="'768px'">
+    <ng-template pTemplate="caption">
+      <div class="flex">
+        <button pButton label="Clear" class="p-button-outlined" icon="pi pi-filter-slash"></button>
+        <span class="p-input-icon-right">
+          <i class="pi pi-search" (click)="busca()"></i>
+          <input type="text" (change)="busca()" [placeholder]="'Pesquisar paciente'" [(ngModel)]="nomePaciente" pInputText/>
```

Comment:
```suggestion
<input class="w-full" type="text" (change)="busca()" [placeholder]="'Pesquisar paciente'" [(ngModel)]="nomePaciente" pInputText/>
```

Assim, o campo não vai expandir mais do que a largura da tabela/página quando estiver pequena.